templates/smart-start/contact-us.html

```diff
@@ -0,0 +1,17 @@
+{% extends "smart-start/base_smart-start.html" %}
+
+{% block title %}Contact us | SMART START{% endblock %}
+
+{% block content %}
+
+{% with h1='Contact us about SMART START',
+  intro_text="Get your IoT project started with SMART START",
+  formid="3591",
+  lpId="",
+  side1="shared/forms/_desktop_help.html",
```

Comment:
I don't think you want either `side1` or `side2`

Reply:
Done